### PR TITLE
Add notebook feature test with behave

### PR DESCRIPTION
**Pull request by ethanwharris**
# Before submitting

- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning-hpo/blob/main/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure to update the docs?
- [ ] Did you write any new necessary tests?

## What does this PR do?

Fixes # (issue).

## PR review

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?

Make sure you had fun coding 🙃

### CODE REVIEW

**Review by tchaton — APPROVED** [review]
Super neat !

File: .github/workflows/ci-tests.yml [hunk 1]
```diff
@@ -82,6 +87,8 @@ jobs:
         if: success()
         working-directory: ./
```
Comment (Borda) [inline .github/workflows/ci-tests.yml:88]: this is default

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -48,7 +48,7 @@ repos:
         name: Check PEP8
         args:
           - "--max-line-length=120"
-          - "--ignore=E402,E501"
+          - "--ignore=E402,E501,F811"
```
Comment (Borda) [inline .pre-commit-config.yaml:51]: shall this in setup.cfg or pyproject.toml?

File: MANIFEST.in [hunk 3]
```diff
@@ -12,5 +13,6 @@ recursive-include requirements *
 
 recursive-exclude docs *
 recursive-exclude tests *.py
+recursive-exclude tests *.feature
```
Comment (Borda) [inline MANIFEST.in:16]: Where this comes from?

File: tests/features/environment.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+            context.page = context.browser.new_page()
+
+            while True:
+                try:
+                    time.sleep(5)
+                    context.page.goto(get_app_url(RuntimeType.MULTIPROCESS))
+                    locator = context.page.frame_locator("iframe").locator("text=Training Studio")
+                    expect(locator).to_be_visible(timeout=30 * 1000)
+                    break
+                except:  # noqa: E722
```
Comment (Borda) [inline tests/features/environment.py:55]: `except Exception:`

File: tests/features/notebook.feature [hunk 5]
```diff
@@ -0,0 +1,42 @@
+Feature: Start Notebook
```
Comment (Borda) [inline tests/features/notebook.feature:1]: can we add some readme for test or/and link for more reading about scenarios?

**Review by Borda — COMMENTED** [review]
The scenarios look cool!

**Pull request merged**